src/tonic/api/points_api.rs

```diff
@@ -71,7 +72,7 @@ impl Points for PointsService {
         let access = extract_access(&mut request);
 
         delete(
-            self.dispatcher.toc(&access).clone(),
+            StrictModeCheckedTocProvider::new(&self.dispatcher),
```

Comment:
I don't really like that we have to color all gRPC functions and pass in a `StrictModeCheckedTocProvider` or the unchecked variant.

But I don't think there is a nice alternative is there?

Reply:
The only alternative I can think of is to have each function twice, checked and unchecked. But I think the current approach with `StrictModeCheckedTocProvider` is much better.